sandboxed_containers/sandboxed-containers-4-8-release-notes.adoc

```diff
@@ -0,0 +1,128 @@
+[id="sandboxed-containers-4-8-release-notes"]
+= {sandboxed-containers-first} 1.0 release notes
+include::modules/common-attributes.adoc[]
+:context: sandboxed-containers-release-notes
+
+toc::[]
+
+[id="ocp-1-0-about-this-release"]
+== About this release
+
+These release notes track the development of {sandboxed-containers-first} in Red Hat {product-title}.
+
+This product is currently in Technology Preview. {sandboxed-containers-first} is not intended for production use. For more information, see the Red Hat Customer Portal link:https://access.redhat.com/support/offerings/techpreview[support scope] for features in Technology Preview.
+
+[id="sandboxed-containers-1-0-new-features-and-enhancements"]
+== New features and enhancements
+
```

Comment:
Do you think it makes sense to include the same line under this header as in the main rel notes?
> This release adds improvements related to the following components and concepts.

Reply:
I actually do think that's a good addition. I'll put it in there :)

Reply:
@jeana-redhat I do, however, 4.8 was the initial release for Sandboxed containers. Do you think it makes sense to add that line there if it's the first release and not an improvement on something that already exists? Happy to add it, if so.

Reply:
Good point! I suppose hand on to it for next release :slightly_smiling_face: